src/RokuDeploy.ts

```diff
@@ -329,7 +329,9 @@ export class RokuDeploy {
             //`pattern` is some other glob magic
         } else {
             const fileNameAndExtension = path.basename(srcPath);
-            result = util.standardizePath(`${entry.dest ?? ''}/${fileNameAndExtension}`);
+            const foundFilePath = util.standardizePath(`${entry.dest ?? ''}/${fileNameAndExtension}`);
+            const assumedFilePath = util.stringReplaceInsensitive(srcPath, rootDir, '');
+            result = entry.dest ? foundFilePath : assumedFilePath;
```

Comment:
This will result in evaluating both foundFilepath and assumedFilepath every time, which is a little CPU wasteful. Let's do an `if` statement or a ternary expression instead:

```suggestion
            if (entry.dest) {
                result = util.standardizePath(`${entry.dest ?? ''}/${fileNameAndExtension}`);
            } else {
                result = util.stringReplaceInsensitive(srcPath, rootDir, '');
            }
```